scripts/tokenize_corpus.py

```diff
@@ -0,0 +1,99 @@
+#
+# Pyserini: Reproducible IR research with sparse and dense representations
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import argparse
+import multiprocessing
+import json
+import time
+from joblib import Parallel, delayed
+from transformers import AutoTokenizer
+from ltr_msmarco.convert_common import get_retokenized
+
+"""
+add fields to jsonl with contents((BERT token)
```

Comment:
inaccurate...

Reply:
I have changed it into "Replace original contents fields with bert tokenization". Is it more accurate?

Reply:
yup